apps/storybook/src/Slider.stories.tsx

```diff
@@ -189,3 +219,18 @@ DecimalIncrement.args = {
   step: 2.5,
   values: [25],
 };
+
+export const Vertical: Story<SliderProps> = (args) => {
+  return <Slider {...args} />;
+};
+
+Vertical.args = {
+  ...Basic.args,
```

Comment:
`Basic.args` only has one value, so just hardcode it here as it will be easier for the user to see.

Reply:
**[DONE]** Right, that makes sense